heron/scheduler-core/src/java/com/twitter/heron/scheduler/SubmitterMain.java

```diff
@@ -315,18 +316,18 @@ public static void main(String[] args) throws Exception {
                 topologyPackage, topologyBinaryFile, topologyDefnFile, topology))
             .build());
 
-    LOG.fine("Static config loaded successfully ");
+    LOG.fine("Static config loaded successfully");
     LOG.fine(config.toString());
 
     SubmitterMain submitterMain = new SubmitterMain(config, topology);
-    boolean isSuccessful = submitterMain.submitTopology();
-
-    // Log the result and exit
-    if (!isSuccessful) {
+    try {
+      submitterMain.submitTopology();
+      // SUPPRESS CHECKSTYLE IllegalCatch
+    } catch (Exception e) {
+      System.out.println(e.getMessage());
```

Comment:
it looks odd to catch Excetption, log only the message to standard our, and than throw a RuntimeException. Please add comments explaining why we're doing this.